src/aliases/api/handlers/handlers_test.go

```diff
@@ -222,7 +222,7 @@ func TestGetAlias(t *testing.T) {
 		c := defaultCase()
 		c.key = nonExistingKey
 		c.status = http.StatusNotFound
-		ent := newEntAlias(c.reg, c.key, "")
+		ent := newEntAlias(c.reg, c.key, nil)
```

Comment:
is this a faker function? In this case it should be placed in a folder called `testutils` and be called `FakeAlias`. Check the other testutils from `stores` domain for example.

Reply:
I will put it in testutils, but I believe it would make more sense in a `testdata` folder which will make any public faker functions available from tests written in any package, without leaking it in the rest of the application. I will add this nuance in the architecture doc.

Reply:
Good one, I didn't know about this standard